%!TEX root = ../dissertation.tex
\chapter{General Conclusion}
\label{conclusion}

The present thesis comprises research using a multitude of approaches
from comparative genomics, ancestral reconstruction, phylogenetics, as
well as algorithm design and implementation. By using a standardized TE
annotation and a large taxon sampling that encompasses representatives
from all major insect orders, it is able to draw conclusions that
surpass inferences from intra-ordinal comparisons. Additionally, the
development of a new software tool enables researchers to easily assess
orthology within coding nucleotide data.

The software Orthograph (chapter \ref{cha:orthograph} has proven to be a
valuable asset which was used in a number of publications. Many of these
aim to resolve order-level or family-level phylogenies from
transcriptomic data (\eg, Hymenoptera: Vespidae \citep{Bank2017},
Hymenoptera \citep{Peters2017}, Diptera: Acroceridae \citep{Gillung2018},
Hemiptera \citep{Johnson2018}, Diptera \citep{Kutty2018}, Palaeoptera
\citep{Simon2018}). In other publications, it was used to map target
enrichment data to the correct genes \citep{Mayer2016, Sann2018,
Shin2018}, often also for phylogenetic analyses. However, Orthograph was
designed to be versatile, and this shows in its application in studies
that investigate the evolution of specific gene families
\citep{Pauli2016, Dowling2017} or the distribution of DNA methylation in
insects \citep{Provataris2018}. Orthograph was reviewed in
\citet{Nichio2017} and has recieved critical acclaim.

By providing, with Orthograph, a powerful and easy to use tool to
identify orthologs in coding nucleotide data, the efforts described in
chapter \ref{cha:orthograph} facilitate future phylogenetic analyses.
The growing amount of available transcriptomic data for more and more
species enables researchers to further resolve phylogenies based on
molecular datasets with increasing resolution and accuracy. Although for
many species, molecular data will never be obtained (see page
\pageref{mass-extinction}), our understanding of the remaining species'
relationships will develop. This endeavor continues to be important
since all studies dealing with aspects of evolution --- also the ones in
this thesis --- necessitate a concept of species interrelationships.
Thus, Orthograph contributes to furthering the field of biological
(molecular) systematics, which in turn enables other fields such as
evolutionary biology or comparative genomics to make meaningful
inferences.

The focus of the empirical comparative studies in this thesis was on
insect genomes. Insects are very different from vertebrates, plants,
and fungi in both phenotype and genotype. This is also reflected in the
processes that define their genome size with respect to the TE content.
As in vertebrates, TE content is a predictor for genome size, however,
DNA gain and loss rates do not affect genome size in insects as much as
they do in vertebrates \citep{Kapusta2017-1, Lindblad-Toh2005}.
Nevertheless, insect genome size does exhibit large fluctuations
\citep{Alfsnes2017}, but these cannot be explained by differential TE
activity alone.  Similarly, the patterns of DNA methylation, which has
been hypothesized to be involved in TE defense mechanisms, in insect
genomes are drastically different from what has been observed in
vertebrates or plants \citep{Provataris2018, Suzuki2008}. Apparently,
insects do not rely on DNA methylation to inhibit TE proliferation and
thereby genome size expansion. What else, then, could explain the large
spread in insect genome size?

The answer to that question is likely not straigthforward, and the
present thesis does not have it. It does, however, provide a broad array
of pointers for future research. For example, the RNA interference
pathway genes were implicated in TE inhibition \citep{Aravin2001,
Czech2008} and are absent in some butterfly species that exhibited high
TE content \citep{Dowling2017}. The large number of publicly available
lepidopteran genomes provides ample opportunity to closely investigate
these genes and shed more light on TE defense mechanisms. Incorporating
some of these genomes, the study in chapter \ref{cha:mobilome}
characterizes the TE repertoire of 73 arthropod species, the largest
taxon sampling for a comparative study on TE diversity in arthropods to
date. The TE annotation data that were generated for the study are
valuable as a resource to investigate the interaction of TEs with other
genome components such as protein-coding genes. The annotation results
from six insect species \todo{how many did Jeanne use?} were used in
investigations on the protein-coding gene repertoire in insects
\todo{how to cite Jeanne?}. Another part of the TE annotation data was
used by \citep{Provataris2018}, and the annotation pipeline was used to
identify TEs in additional insect genomes. The annotation procedure is
largely identical to the one used by \citet{Reinar2016} who benchmarked
the approach and showed it to be accurate and efficient. By combining
several well-established algorithm implementations into an easy to use
and fully automated pipeline, the shell script provides a tool to
reliably annotate TEs in assembled genomes of non-model species. 

The construction of a dated phylogeny for over 600 insect and arthropod
species from the literature and publicly available DNA barcode data is
unprecendented (chapter \ref{cha:dynamics}). This phylogeny will be
valuable to researchers in many disciplines because it allows to set
insights from other studies into context with the evolution of genome
size in insects.  In fact, the phylogeny also allows to map other
phenotypic characters and to infer ancestral states for them, which is
often a means to study and understand their evolution.

Also in chapter \ref{cha:dynamics}, I inferred ancestral genome sizes
for 613 arthropod taxa including 520 insect species using that dated
phylogeny and likewise publicly available genome size data
\citep{Gregory2018}. While other studies have also exploited this
database to set extant insect genome size into context with other
phenotypic traits \citep{Alfsnes2017, Gregory2011}, none had information
on the ancestral states of these traits due to lack of a phylogeny with
branch lengths. Using the obtained ancestral genome size estimates, it
was possible to classify the annotated TE content into ancestral and
lineage-specific TEs. The study shows that there are practically no
ancestral TEs in arthropod species that diverged from the common
ancestor of their sister species earlier than about 100 Mya. This result
is consistent with prior findings that inactive TEs become
unrecognizable after more than 50 Mya due to random mutations
\citep{Shedlock2000} and leads to the hypothesis that the majority of
TEs in extant genomes might be dormant and possibly suppressed by the
host genome defenses such as the RNAi or piRNA pathways or DNA
methylation. Save from sustaining a high gene deletion rate (along with
its drawbacks), no mechanism for targeted removal of TEs has been
identified in eukaryotes. Thus, the most efficient defense appears to be
to reduce TE activity and let random mutation degrade the TEs. Only
during a period of inefficient silencing, for example due to relaxed
epigenetic modification, would the TEs be able to successfully
proliferate \citep{Slotkin2007, Zeh2009, Rebollo2010}, leading to a
burst in TE activity as often observed in the study in chapter
\ref{cha:mobilome}.  These periods of epigenetic silencing could be
caused by environmental stress \citep{Horvath2017, Horvath2017-1}, an
opportunity for adaptive evolution to work.

In general, the role of TEs in adaptive evolution cannot be disregarded.
After decades of being viewed as mainly deleterious or neutral in effect
on the host genome, the reputation of TEs changed when evidence for
beneficial functions conferred by TEs was discovered (reviewed in
\citet{Oliver2012, Fedoroff2013}). Especially in times of stress, when
the organism is in need of genomic innovation to survive and adapt to
new environmental conditions, TEs are thought to play an important role.
For instance, TEs have been implicated in the rewiring of regulatory
networks conferring dosage compensation \citep{Ellison2013, Chuong2017}
or in adaptation to a different climate \citep{Gonzalez2010}.
Additionally, there is no difference in the ratio of beneficial to
deleterious TE-derived mutations when compared to mutations caused by
single nucleotide polymorphisms (SNPs) \citep{Akagi2013, Barron2014}.
Therefore, the rate of beneficial or destructive effects due to TE
activity is no different than that of random nucleotide substitutions,
however, the effects are more profound when they are caused by TEs
because they affect a larger region of the genome and can cause
chromosomal rearrangements due to ectopic recombination \citep{Gray2000,
Fiston-Lavier2007}. In \species{Drosophila melanogaster} and \species{D.
miranda}, which exhibit similar rates of adaptation \citep{Bachtrog2008}
(\species{Drosophila melanogaster} also shows a high rate of TE-induced
adaptation \citep{Gonzalez2008}), the study in chapter
\ref{cha:mobilome} inferred a two-fold difference in TE coverage. This
is only an apparent contradiction, however: \species{D. miranda}
exhibits a smaller current population size \citep{Bachtrog2008}, where
the impact of genetic drift is amplified. The rate of fixation of a
mutation is also higher in small populations \citep{Kimura1969}, thus it
is not surprising that a lower TE content in \species{D. miranda}, as
found in chapter \ref{cha:mobilome}, is not reflected in a lower rate of
adaptation.

What to do with these?

\begin{itemize}
\item In a study on nematode genomes, \citet{Szitenberg2016} argue that
long-term TE dynamics are largely independent of host genome defenses,
and that TE evolution in the host genome is determined by genetic drift.

\item The findings in chapter \ref{cha:dynamics} are better explained by the
burst model than by the equilibrium model of TE dynamics. 
\end{itemize}

\section{Roadmap}

find a way back to the beginning
