%!TEX root = ../dissertation.tex
\chapter{General Conclusion}
\label{conclusion}

The present thesis comprises research using a multitude of approaches
from comparative genomics, ancestral reconstruction, phylogenetics,
algorithm design and implementation. By using a standardized TE
annotation and a large taxon sampling that encompasses representatives
from all major insect orders, it is able to draw conclusions that
surpass inferences from intra-ordinal comparisons.  Additionally, the
development of a new software tool enables researchers to flexibly
assess orthology within coding nucleotide data.

The focus of the empirical studies in this thesis was on insect genomes.
Insects are very different from vertebrates and plants in both phenotype
and genotype. This is also reflected in the processes that define their
genome size with respect to the TE content. As in vertebrates, TE
content is a predictor for genome size, but the similarity seems to end
there.  DNA gain and loss rates do not affect genome size in insects as
much as they do in vertebrates \citep{Kapusta2017-1,Lindblad-Toh2005}.
Nevertheless, insect genome size does exhibit large fluctuations
\citep{Alfsnes2017}, but these cannot be explained by differential TE
activity alone. Similarly, the patterns of DNA methylation, which has
been hypothesized to be involved in TE defense mechanisms, in insect
genomes are drastically different from what has been observed in
vertebrates or plants \citep{Panagiotis2018,Suzuki2008}.  Apparently,
insects do not rely on DNA methylation to inhibit TE proliferation and
thereby genome size expansion. What else, then, could explain the large
spread in insect genome size?

The answer to that question is likely not straigthforward, and the
present thesis does not have it. It does, however, provide pointers for
future research. For example, the RNA interference pathway genes were
implicated in TE inhibition \citep{Aravin2001,Czech2008} and are absent
in some butterfly species that exhibited high TE content
\citep{Dowling2017}.  The large number of available lepidopteran genomes
provides ample opportunity to closely investigate these genes and shed
more light on TE defense mechanisms.  The TE annotation data that were
generated for the study in chapter \ref{cha:mobilome} are valuable as a
resource to study the interaction of TEs with other genome components
such as protein-coding genes. The annotation procedure is largely
identical to the one used by \citet{Reinar2016} who benchmarked the
approach and showed it to be accurate and efficient. The annotation
results from six insect species \todo{how many did Jeanne use?} were
used in investigations on the protein-coding gene repertoire in insects
\todo{how to cite Jeanne?}. Another part of the TE annotation data was
used by \citep{Provataris2018}, and the annotation pipeline (which might
be published separately at a later time) was used to identify TEs in
additional insect genomes. By combining several well-established
algorithm implementations into an easy to use and fully automated
pipeline, the shell script provides a tool to reliably annotate TEs in
assembled genomes of non-model species. 

The construction of a dated phylogeny for over 600 insect and arthropod
species from the literature and publicly available DNA barcode data is
unprecendented and allowed the inference of ancestral genome sizes
(chapter \ref{cha:dynamics}). This phylogeny will be valuable to
researchers in many disciplines because it allows to set insights from
other studies into context with the evolution of genome size in insects.
In fact, the phylogeny also allows to map other phenotypical characters
and to infer ancestral states for them, which is often a means to study
and understand their evolution.

\section{Roadmap}

Orthograph (chapter \ref{cha:orthograph}) provides versatile tool ---
not only for phylogenetics but also for other disciplines --- has been
used in many studies --- mostly phylogenetics, but also evolutionary
biology (GÃ³mez-Zurita) --- not limited to transcriptomes but also useful
with HE data --- etc --- phylogenetics important because provides the
backbone phylogeny that is required for all evolutionary inferences ---
in chapter \ref{cha:mobilome} the insect mobilome is characterized ---
etc
