%!TEX root = ../dissertation.tex
\chapter{General Conclusion}
\label{conclusion}

The present thesis comprises research using a multitude of approaches
from comparative genomics, ancestral reconstruction, phylogenetics,
algorithm design and implementation. By using a standardized TE
annotation and a large taxon sampling that encompasses representatives
from all major insect orders, it is able to draw conclusions that
surpass inferences from intra-ordinal comparisons.  Additionally, the
development of a new software tool enables researchers to flexibly
assess orthology within coding nucleotide data.

The software Orthograph has proven to be a valuable asset which was used
in a number of publications. Many of these aim to resolve order-level
phylogenies from transcriptomic data (\eg, \citet{Peters2017, Bank2017,
Kutty2018, Simon2018, Gillung2018, Johnson2018}). Orthograph is not only
limited to transcriptomes, however: In other publications, it was used
to map target enrichment data to the correct genes \citep{Mayer2016,
Shin2017, Sann2018}, often also for phylogenetic analyses. Orthograph
was designed to be versatile, and this shows in its application in
studies that investigate the evolution of specific gene families
\citep{Pauli2016, Dowling2017} or the distribution of DNA methylation in
insects \citep{Provataris2018}. Orthograph was reviewed in
\citet{Nichio2017} and has recieved critical acclaim.

By developing Orthograph, I have contributed to facilitate phylogenetic
analyses for researchers. While the backbone phylogeny for insects has
been resolved \citep{Misof12014}, many systematic questions on order and
family level relationships remain. A growing number of studies have
attempted to resolve them, but a comprehensive insect phylogeny, at
least on genus level, has not been proposed. Of course, this is also due
to the sheer number of insect species (see Figure \ref{fig:biodiversity}
on page \pageref{fig:biodiversity}) that would require monumental
sampling efforts.

The focus of the empirical studies in this thesis was on insect genomes.
Insects are very different from vertebrates, plants, and fungi in both phenotype
and genotype. This is also reflected in the processes that define their
genome size with respect to the TE content. As in vertebrates, TE
content is a predictor for genome size, however,  DNA gain and loss
rates do not affect genome size in insects as much as they do in
vertebrates \citep{Kapusta2017-1,Lindblad-Toh2005}.  Nevertheless,
insect genome size does exhibit large fluctuations \citep{Alfsnes2017},
but these cannot be explained by differential TE activity alone.
Similarly, the patterns of DNA methylation, which has been hypothesized
to be involved in TE defense mechanisms, in insect genomes are
drastically different from what has been observed in vertebrates or
plants \citep{Provataris2018,Suzuki2008}.  Apparently, insects do not
rely on DNA methylation to inhibit TE proliferation and thereby genome
size expansion. What else, then, could explain the large spread in
insect genome size?

The answer to that question is likely not straigthforward, and the
present thesis does not have it. It does, however, provide pointers for
future research. For example, the RNA interference pathway genes were
implicated in TE inhibition \citep{Aravin2001,Czech2008} and are absent
in some butterfly species that exhibited high TE content
\citep{Dowling2017}.  The large number of available lepidopteran genomes
provides ample opportunity to closely investigate these genes and shed
more light on TE defense mechanisms.  The TE annotation data that were
generated for the study in chapter \ref{cha:mobilome} are valuable as a
resource to study the interaction of TEs with other genome components
such as protein-coding genes. The annotation procedure is largely
identical to the one used by \citet{Reinar2016} who benchmarked the
approach and showed it to be accurate and efficient. The annotation
results from six insect species \todo{how many did Jeanne use?} were
used in investigations on the protein-coding gene repertoire in insects
\todo{how to cite Jeanne?}. Another part of the TE annotation data was
used by \citep{Provataris2018}, and the annotation pipeline (which might
be published separately at a later time) was used to identify TEs in
additional insect genomes. By combining several well-established
algorithm implementations into an easy to use and fully automated
pipeline, the shell script provides a tool to reliably annotate TEs in
assembled genomes of non-model species. 

The construction of a dated phylogeny for over 600 insect and arthropod
species from the literature and publicly available DNA barcode data is
unprecendented (chapter \ref{cha:dynamics}). This phylogeny will be
valuable to researchers in many disciplines because it allows to set
insights from other studies into context with the evolution of genome
size in insects.  In fact, the phylogeny also allows to map other
phenotypic characters and to infer ancestral states for them, which is
often a means to study and understand their evolution.

Also in chapter \ref{cha:dynamics}, I inferred ancestral genome sizes
for 613 arthropod taxa including 520 insect species using that dated
phylogeny and likewise publicly available genome size data
\citep{Gregory2018}. While other studies have also exploited this
database to set extant insect genome size into context with other
phenotypic traits \citep{Alfsnes2017, Gregory2011}, none had information
on the ancestral states of these traits due to lack of a phylogeny with
branch lengths. Using the obtained ancestral genome size estimates, it
was possible to classify the TE content into ancestral and
lineage-specific. The study shows that there are practically no
ancestral TEs in arthropod species that diverged from the common
ancestor of their sister species earlier than about 100 Mya. This result
is consistent with prior findings that inactive TEs become
unrecognizable after more than 50 Mya due to genetic drift
\citep{Shedlock2000}. This result leads to the hypothesis that the
majority of TEs in extant genomes might be dormant and possibly
suppressed by the host genome defenses. Only during a period of
inefficient silencing would the TEs be able to successfully proliferate,
leading to a burst in TE activity as often observed in the study in
chapter \ref{cha:mobilome}. These findings are better explained by the
burst model than by the equilibrium model of TE dynamics.

\section{Roadmap}

Orthograph (chapter \ref{cha:orthograph}) provides versatile tool ---
not only for phylogenetics but also for other disciplines --- has been
used in many studies --- mostly phylogenetics, but also evolutionary
biology (GÃ³mez-Zurita) --- not limited to transcriptomes but also useful
with HE data --- etc --- phylogenetics important because provides the
backbone phylogeny that is required for all evolutionary inferences ---
in chapter \ref{cha:mobilome} the insect mobilome is characterized ---
etc
