%!TEX root = ../dissertation.tex
\chapter{General Introduction}
\label{general-introduction}

\newthought{The genome, that is, the entirety of DNA of an organism} is
a composition of different functional complexes. It does not only
contain genes, which are transcribed to messenger RNA and translated
into the proteins that make up cells and, ultimately, all organisms; in
fact, the human gene repertoire of around 23,000 genes makes up only
around \p{2} of the human genome \citep{Makalowski2001} (Figure
\ref{fig:human-genome}). More prominent components of the human genome
include introns (non-coding sections of genes, around \p{26}), but by
far the most voluminous chunk consists of repetitive elements: DNA
segments that occur in sometimes many copies throughout the genome.
More than half of the three billion base pairs (Gbp) of the human genome
(\p{52}) is occupied by repetitive elements \citep{Lander2001,deKoning2011}.
The major part of these repetitive elements in the human genome, also called
repeats, is formed by transposable elements (\p{45} of the genome).

Transposable elements (TEs) are also known as ``jumping genes'' or
``parasitic DNA''. They were discovered in the 1940s by their defining
property, the capability of movement within the genome
\citep{McClintock1950}. By duplicating themselves through various
mechanisms that depend on the TE type, TEs can reach copy numbers in the
thousands \citep{Petersen2018} and, like in the human genome (Figure
\ref{fig:human-genome}), be a major contributor to the genome size. This
genome ``inflation'' effect due to TE proliferation has been observed
throughout eukaryotes in general \citep{Chenais2012}, and reiterated in
vertebrates \citep{Chalopin2015}, arthropods \citep{Petersen2018}, and
plants \citep{Staton2015}. In contrast, there are species with small
genomes that carry a small TE load. This has been observed in plants
\citep{Ibarra-Laclette2013}, nematodes \citep{Burke2015}, and insects
\citep{Kelley2014}. 

\begin{figure}
	\centering
  \begin{minipage}[c]{0.3\textwidth}
		\caption[Composition of the human genome]{Composition of the human
		genome. Almost half of the three billion base pairs in the genome is
		attributed to transposable elements of various classes (DNA transposons,
		LTR retrotransposons, LINEs, SINEs). Data source: \citet{Lander2001}}
		\label{fig:human-genome}
  \end{minipage}\hspace{3em}
  \begin{minipage}[c]{0.3\textwidth}
		\includegraphics[width=\textwidth]{human}
  \end{minipage}
\end{figure}

The genomes of mammals, such as human, and birds exhibit much less
variation in size than, for example, the genomes of arthropods or
amphibians \citep{Gregory2005}. In mammals, genome size varies around
five-fold and in birds even only around two-fold, whereas in insects,
the spread is around 240-fold (Figure \ref{fig:genome-size-spread},
Table \ref{tab:genome-size-spread} on page
\pageref{tab:genome-size-spread}). This immense variation surpasses that
of amphibians, where some species have huge genomes of up to 118 Gbp,
and is paralleled only by the group of bony fishes (Osteichthyes,
excluding lungfishes), which exhibit a genome size spread of around
220-fold. Before the discovery of TEs and non-coding DNA, such as
introns, in the genome, it was assumed that genome size should correlate
with perceived organismic complexity, but the fact that amoeba have
genomes with up to a staggering 670 Gbp \citep{Parfrey2008} did not fit
well with that assumption. This apparent contradiction was named the
``C-value paradox'' and later renamed to ``C-value enigma''
\citep{Gregory2007}, as still a connection between genome size and
organismic complexity appears absent.

\begin{figure}
\centering
\includegraphics[width=0.7\textwidth]{genome-size-spread}
\caption[Genome size spread in eukaryotes and prokaryotes]{Genome size
spread in eukaryotes and prokaryotes. The C-value is the amount of
haploid nuclear DNA in picogram (pg); one pg DNA is approximately 978
Mbp. Colors are ordered for chordate animals, invertebrate animals,
plants, fungi and algae, and prokaryotes.  Figure modified from
\citet{Gregory2004}.}
\label{fig:genome-size-spread}
\end{figure}

No matter the size, the genome needs to be maintained: repair mechanisms
and transcription machinery as well as error correction use energy. The
transcription and translation error rate increases with genome size
\citep{Zaher2009}, making more repairs necessary. Larger genome size has
been linked to decreased development rate \citep{White2000} and
increased oxygen requirements \citep{Vinogradov1997, Gregory2002}. In
plants \citep{Grime1983}, invertebrates \citep{Gregory2005}, and
vertebrates \citep{Horner1983, Olmo1982, Gregory2000}, it has been shown
that cell size increases with genome size \citep{Dufresne2011}. Larger
cells are also less efficient to maintain and proliferate, and they
divide more slowly \citep{Bennett1977}. In summary, a large genome comes
with a cost. What would be the benefit of having a large genome?

The more genetic material in the genome, the higher the likelihood of
random mutations \citep{Wielgoss2011}. Mutations provide the basis for
genotypic evolution: through natural selection, deleterious changes to
the genome will be removed from populations over time, and beneficial
changes --- or innovations --- prevail. A higher mutation rate is not
always a negative property: it also brings with it a higher rate of
beneficial mutations. Therefore, the genome is thought to reach an
equilibrium between the incurred metabolic cost of sustaining a high
mutation rate (\emph{i.e.}, the damage caused by deleterious mutations),
and the cost of mechanisms that reduce the mutation rate
\citep{Bernstein1987, Altenberg2011}.

\section{Impact of transposable elements on the genome}

The presence and activity of TEs can have disruptive influence on the
genome architecture. By inserting at critical positions, TEs can disable
genes \citep{Kazazian1988}. An insertion in regulatory sequence can
change gene expression \citep{Warnefors2010}. TEs, by way of their
repetitive nature, provide hotspots for ectopic (non-homologous)
recombination \citep{Lim1988, Gray2000, Fiston-Lavier2007}, thus
increasing the likelihood for segmental duplications, deletions, and
inversions \citep{Mathiopoulos1998, Remnant2013}. On the one hand, TEs
are obviously a source of potentially deleterious mutations. On the
other hand, TEs can be ``domesticated'' and genes exapted from TE
sequence \citep{Gahan2001, Daborn2002, Aminetzach2005, Chen2007},
conferring novel functions to the host. Such innovations can happen
within a few hundred generations \citep{Dolgin2006, Struchiner2009,
Kofler2015}. As a famous example, the melanism in the British peppered
moth --- in which camouflage evolved that matches the birch trees
blackened as a result of industrialisation --- is caused by TEs
\citep{Hof2016}. These observations document that TE activity can also
have beneficial effects on the host genome (especially in times of
stress \citep{Chenais2012}), and should therefore not be entirely
subdued.

To keep the TE population in check, defenses that remove or silence TEs
have developed in host organisms. In many groups of organisms, a
multi-layered network of epigenetic regulation mechanisms evolved in
place to prevent TE activity at both the pre- and post-transcriptional
stage. In plants, an epigenetic modification called DNA methylation
prevents TEs from being transcribed and thus from transposing
\citep{Slotkin2007, Lisch2009}. After transcription, proteins from the
RNA interference (RNAi) pathway can disable messenger RNA and thereby
silence TEs \citep{Buchon2006}. Similarly, a class of non-coding RNA,
so-called Piwi-interacting RNA (piRNA) protect the integrity of the
genome, in particular in germline cells, by forming a complex with Piwi
proteins, which can bind and cleave RNA \citep{Zeng2011}. This complex
can recognize and silence target TEs in the RNA stage \citep{Siomi2011,
Mondal2018}.  Similar systems were identified in vertebrate genomes
\citep{Suzuki2008, Schubeler2015}: DNA methylation is thought to be a
genome defense mechanism in mammals as well \citep{Yoder1997}.
Interestingly, vertebrate genomes are globally methylated, and in plant
genomes, only gene bodies and TEs are methylated \citep{Suzuki2008}.
Fungal genomes exhibit an even more mosaic-like methylation pattern:
here, only TEs are methylated and genes are not. In invertebrates, TEs
tend to be unmethylated. The fruit fly \species{Drosophila melanogaster}
does not even have the methyltransferase enzyme in its gene repertoire. Likewise, some
butterfly species have lost RNAi pathway genes \citep{Pauli2016}. Thus,
these genome defenses appear to be modular and complementary to one
another. They are effective to a certain extent: permanently inactive
TEs become genomic ``cruft'' and are degraded by random mutations over
time and genetic drift like other parts of the genome that are not
subject to selection \citep{Szitenberg2016}. As a result of these
extensive silencing techniques, it is not surprising that most of the TE
population in extant genomes is inactive \citep{Yoder1997,
Zilberman2007}.

There are two major models to explain TE population dynamics in the
genome: the equilibrium model and the burst model \citep{Petrov2011,
Kofler2012, Cridland2013, Blumenstiel2014}. In the equilibrium model,
the TE insertion rate is assumed to be more or less constant, and TEs
are silenced and removed by purifying selection at a likewise constant
rate \citep{Charlesworth1983}. This way, TE insertion rate and DNA
removal rate would cancel each other out, and the genome size remains
stable.  The equilibrium model provides a better fit for TE dynamics
under the effects of purifying selection \citep{Barron2014} than the
transposition burst model. The burst model, which is also termed the
non-equilibrium model, predicts that TEs undergo periods of high
transposition activity while otherwise proliferating at a constant but
lower rate. Under the transposition burst model, a positive correlation
between the TE age and frequency would be expected, which better
explains the observed TE age distribution in insect genomes as well as
the large genome size fluctuations during insect evolution, given that
TE abundance is a predictor for genome size \citep{Alfsnes2017,
Petersen2018a}.

\section{Insects, phylogenetics, and comparative genomics}

Insects are among the most speciose groups of organisms on earth (Figure
\ref{fig:biodiversity} on page \pageref{fig:biodiversity}) and, since
their appearance approximately 480 million years ago (Mya)
\citep{Misof2014}, have conquered land, freshwater, and air (but not
saltwater). Protected by their hard exoskeleton, insect representatives
have invaded virtually all conceivable ecosystems including human
habitations \citep{Bertone2016}.  Insects are immensely diverse in
morphology \citep{Grimaldi2005} and often highly specialized towards a
specific food source, habitat, or lifestyle. Bees, wasps, ants, and
termites, for example, form eusocial communities with a complex caste
system. As disease vectors, mosquitoes are responsible for more human
deaths than all other animals combined \citep{WHO2017, Linnell2011,
Lamarque2009, DeMaddalena2008, Kasturiratne2008, Packer2005}. Beetles,
cicadas, and grasshoppers are examples for an important source of food
for livestock and humans alike as well as a pest with high economic
impact \citep{Oliveira2014}. Obviously, insects play pivotal roles in
most ecosystems of the planet. Insect population diversity and
abundance, however, is declining \citep{Vogel2017} as a result of
widespread human influence (see below), with disastrous reverberations
at all levels of the local food chains.  In order to mount efficient
conservation efforts, a thorough understanding of insect biology is
required.

\begin{figure}[t]
\centering
\includegraphics[width=0.75\textwidth]{biodiversity}
\caption[Biodiversity by numbers of species]
{Insects contribute more than half of the currently known species
diversity. Data source: \citet{IUCN2018}; pictures source:
\href{https://pixabay.com}{Pixabay}, available under the Creative
Commons Public Domain license
(\href{https://creativecommons.org/publicdomain/zero/1.0}{\footnotesize{\cczero}}).}
\label{fig:biodiversity}
\end{figure}

Despite their mega-diversity and ecological importance, insects are
astonishingly understudied on the genomic level compared to other
animals: As of 2018-07-06, there were 1,115 published non-insect animal
genome sequences in the NCBI database \citep{OLeary2016} (about \p{0.3}
of the total non-insect species diversity; Figure
\ref{fig:biodiversity}) and 493 insect genomes (\p{0.065}; about five
times less genomes per species).  That number is growing by about 50
genomes per year \citep{OLeary2016}, however at this rate, it will take
more than 150 years to even sequence the genomes of \p{10} of the insect
biodiversity.  During this time, many species will have gone extinct ---
as it stands, our genome sequencing efforts are losing the race against
man-made biodiversity loss. Ever-improving, massively parallel
sequencing technologies that appeared during the early 2000's
\citep{Behjati2013} have accelerated the pace and accuracy at which
genomes can be sequenced, but it will not be enough. Of many insect
species, we will never be able to obtain the genomic source code, and
this will hamper our understanding of the roles these species occupied
in the interaction network of their habitats.

\label{mass-extinction}%
This not only affects insects, but all kingdoms of life: we are
currently experiencing a mass extinction event that parallels other
episodes in earth's history with high rates of biodiversity decline
\citep{Pimm1995, Dirzo2003, Schipper2008, Barnosky2011, Dirzo2014}.
Other than the five previous major extinction events
\citep{Kolbert2014}, it is anthropogenic in origin \citep{Leakey1996,
Ceballos2015} and is associated with global warming \citep{Cook2016,
Wuebbles2017}, large-scale deforestation \citep{Wright2005}, destruction
of marine and freshwater habitats \citep{Burkhead2012}, and introduction
of invasive species \citep{Mooney2001}, all hallmarks of human
influence. Put shortly, the rate at which species go extinct is alarming
\citep{Newbold2016, Ceballos2017, Hallmann2017}, and our children will
likely experience a world with less than half the biodiversity we know
today. While this issue has raised the attention of country leaders and
conservation policies are being put in place worldwide
\citep{Puntaru2017}, this might not be enough to reverse the trend
without sustaining irreparable damage to the ecosystems of the planet.
To make matters worse, there are signs that the issue, despite its
urgency, is fading from public awareness \citep{Kusmanoff2017}. 

We cannot save what we do not know, thus conservation efforts require
intimate knowledge of the systems they aim to preserve. The road towards
understanding the biology and the interaction of species is, however,
travelled on multiple levels. It is not enough to observe the behaviour
or the ecology of an animal to understand the impact of it being removed
from its habitat. It is also not enough to describe functional
morphology to gain insight on ecological implications.  Neither is it
sufficient to analyze the genes and draw conclusions based on their
composition and structure. Profound understanding of any system can only
be gained by studying it from multiple angles and with interdisciplinary
approaches. One approach that can add to the knowledge about a species
is to sequence and analyze its genome: the ``source code of life'' that
defines, by a manifold of means, its appearance, features, behaviour and
interactions with the environment. By comparing the composition, the
functional networks, and other properties of one species' genome to that
of other species, one can gain insights on the mechanisms of evolution
that led to the mega-diversity of today's insects.

Comparative genomics studies --- that is, investigations comparing
genomic features of more than one species --- are not only limited by
the fact that for many species, there is no genomic sequence information
available.  Additionally, comparative analyses have to take the
evolutionary history of the species into consideration \citep{Dunn2018},
usually in the form of a phylogenetic tree that conveys information on
the species' relationships. An undisputed phylogenetic tree down to
family level does not exist for insects so far. \citet{Misof2014} and
the 1KITE project \citep{1KITE2018} have inferred a robust backbone
phylogeny for most major insect orders from transcriptomic data (see
below), and several publications have presented order-level phylogenies,
for example for Coleoptera \citep{McKenna2015}, Hymenoptera
\citep{Peters2017, Branstetter2017}, Lepidoptera \citep{Breinholt2018},
and Hemiptera \citep{Johnson2018}. However, accurately reconstructing
species ancestry remains a challenge that has obstructed reliable
comparative analyses in insects so far.

Among the approaches to infer a phylogenetic species tree, the most
informed, because based on matrices with data points numbering in the
millions, is reconstruction from amino acid or nucleotide sequences.
Phylogenetic species tree reconstruction, however, regardless the
method, is no simple task and relies on sophisticated methods on all
stages of the analysis (proposed by \citet{Misof2014}, for example).
Perhaps most of all, selection of the phylogenetic markers (the features
that distinguish species and define their level of relatedness) is
paramount.  Most modern genomic studies use single-copy genes that are
found in all (or almost all) species. The implied assumption is that
these genes share a common ancestry and are related via speciation
events, that is, they are \emph{orthologous} to one another
\citep{Koonin2005} and their phylogeny reflects the species phylogeny.

Several commonly used methods to identify orthologous genes rely on
clustering nucleotide or amino acid sequences based on their similarity
\citep{Chen2007a}. Since orthologs tend to be more similar to each other
than to all other genes in the genomes under comparison
\citep{Altenhoff2012a}, the orthology hypothesis can be tested via a
bi-directional search for similarity applying a best reciprocal hit
(BRH) criterion: Only if the genes in question form the best search hit
in both directions (\ie, the genes are more similar to each other than
to all other genes) they can be assumed to be orthologous. By grouping
genes from multiple species that share BRH relations, one can form
clusters of orthologous genes or simply orthologous groups (OGs)
\citep{Altenhoff2012}. This approach has been implemented in several
software packages for use in genomic datasets \citep{Li2003,
Tatusov2003, Berglund2008, Zdobnov2017}.  Obtaining a complete and
accurate genome sequence is, however, associated with technical
difficulties and a cost that depends on the genome size. For these
reasons, many phylogenetic studies employ transcriptomes: the nucleotide
sequences of transcripts present in the sample at the time of RNA
fixation \citep{Wang2009}. Transcriptomes can be sequenced at a fixed
cost, however, in contrast to genomes, transcriptomes are inherently
incomplete with regard to the gene set, simply because not all genes are
expressed all the time and may therefore be absent from the sequenced
RNA sample. While this is usually not a problem for phylogenetic
analyses \citep{Wiens2006}, it means that the above-mentioned methods to
\emph{de novo} infer orthology designed for genomes cannot not be used
on transcriptomic data. To infer orthology among transcripts, one
usually applies a reference-based strategy that maps transcripts to
known OGs. In the study by \citet{Misof2014}, we used a software that
implements this approach \citep{Ebersberger2009}, however, during the
analyses, it became obvious that it had several design issues that were
non-trivial to fix.  Therefore, I concepted and wrote a
re-implementation of the reference-based BRH orthology inference
approach that mitigates those issues while delivering equal performance
\citep{Petersen2017}. The software, Orthograph, is described in chapter
\ref{cha:orthograph}.

\begin{table}
\centering\footnotesize
\caption[Genome size spread in Metazoa]{Genome size spread in Metazoa.
Values are in picogram DNA; one pg is approx. 978 mega-basepairs (Mbp).
Data from the Genome Size Database \citep{Gregory2018},
\url{http://www.genomesize.com}, accessed 2018-05-07.}
\label{tab:genome-size-spread}
\begin{tabular}{@{}lllrrrr@{}}
\toprule
Phylum     & Subphylum   & Class          & n    & min  & max   & $\Delta$fold \\
\midrule
Annelida   & --          & Oligochaeta    & 35   & 0.43 & 7.64  & 17.77  \\
Annelida   & --          & Polychaeta     & 100  & 0.06 & 7.2   & 120    \\
Arthropoda & Chelicerata & Arachnida      & 148  & 0.08 & 7.5   & 93.75  \\
Arthropoda & Crustacea   & Branchiopoda   & 68   & 0.16 & 2.91  & 18.19  \\
Arthropoda & Crustacea   & Copepoda       & 73   & 0.14 & 14.68 & 104.86 \\
Arthropoda & Crustacea   & Malacostraca   & 241  & 0.68 & 64.62 & 95.03  \\
Arthropoda & Hexapoda    & Insecta        & 1353 & 0.07 & 16.93 & 241.86 \\
Chordata   & Vertebrata  & Amphibia       & 932  & 0.95 & 120.6 & 126.95 \\
Chordata   & Vertebrata  & Aves           & 903  & 0.91 & 2.16  & 2.37   \\
Chordata   & Vertebrata  & Chondrichthyes & 199  & 1.51 & 17.05 & 11.29  \\
Chordata   & Vertebrata  & Mammalia       & 816  & 1.63 & 8.4   & 5.15   \\
Chordata   & Vertebrata  & Osteichthyes   & 1909 & 0.34 & 74.86 & 220.18 \\
Chordata   & Vertebrata  & Reptilia       & 423  & 1.05 & 5.44  & 5.18   \\
Mollusca   & --          & Bivalvia       & 108  & 0.65 & 5.4   & 8.31   \\
Mollusca   & --          & Gastropoda     & 149  & 0.43 & 7.85  & 18.26  \\
Nematoda   & --          & Secernentea    & 72   & 0.02 & 2.5   & 125    \\
\bottomrule
\end{tabular}
\end{table}

\section{Research questions}

Insects and arthropods differ at the genomic and phenotypic level from
vertebrate representatives. Like all forms of life, they share general
mechanisms of genetic and genomic functionality and universal elements
such as genes and TEs. However, genome composition, architecture and
structural dynamics in insects appear drastically different from
vertebrates and plants. In particular, the insect TE repertoire has not
been subject to a large-scale study that would enable comparisons both
within and between orders. The TE content has been assessed and put into
context with the genome size in studies focusing on mosquitoes
\citep{Neafsey2015} and \species{Drosophila} fruit flies
\citep{Sessegolo2016}), but conclusions across orders are hampered by
absent information on the TE content and composition of key species as
well as by non-standard TE annotation methods that make comparisons
difficult. Additionally, a pan-ordinal taxon sampling could facilitate
the inference of the ancestral insect TE content. 

% hypotheses:
% a) the TE composition and abundance are not random, but
% phylogeny-dependent
% b) TE content is a predictor for genome size

In chapter \ref{cha:mobilome}, I characterize the insect TE repertoire
in a comparative standardized study based on genomic sequence data from
73 insect and non-insect arthropod species encompassing all major
orders. The results highlight differential TE abundance and composition
in comparisons between and within insect orders. The study also
demonstrates that the TE diversity in insects is much larger than
previously thought. The correlation between TE content and genome size
in insects is substantiated. Additionally, the chapter shows that the TE
copy divergence distribution is highly diverse and that some TEs are
recently active among insects.

% Warum? ``Da sind unterschiede zwischen verts und insekten, aber. . .
% weiss man noch nicht und dabei koennte . . . herauskommen''

Genome size in insects is subject to large fluctuations, and TE content
is not only correlated to genome size, but also to the insect phylogeny.
To illuminate the effects of TE activity on genome size dynamics in
insects, chapter \ref{cha:dynamics} presents an extended analysis on the
genomes of 96 arthropod species and genome size estimate data from 613
arthropod species. The study infers a comprehensive, time-calibrated
phylogeny for these species using information from published studies and
mitochondrial nucleotide sequences. This phylogeny is used to infer
ancestral genome sizes and the amount of genome expansion and
contraction since the last ancestor as a result of TE activity in the
genome. It is shown that after about 100 Mya, the amount of ancestral
TEs converges to zero. The rates of TE-associated DNA gain and loss are
correlated with the phylogeny. The chapter also discusses a trend
towards genome size stability despite varying rates of DNA removal in
insects.

In chapter \ref{cha:orthograph}, I present the software Orthograph as a
tool for orthology assessment in transcriptomic --- or other coding
nucleotide sequence --- datasets. Building on and incorporating existing
software packages for sequence similarity search and comparison, it
implements an algorithm that uses the BRH criterion to map transcripts
to clusters of genes with known orthology. The software is written in a
modular fashion to make it flexible and portable. The chapter discusses the
issues that previous implementations of the BRH mapping strategy suffer
from and shows that Orthograph overcomes them by employing a relational
database backend system that enables it to compare millions of search
results in short time. Orthograph represents a powerful, versatile, and
future-proof application which has been used in seven co-authored
studies \citep{Mayer2016, Pauli2016, Bank2017, Dowling2017, Peters2017,
Gillung2018, Johnson2018} that are in appendix
\ref{app:papers-using-orthograph}.

Appendix \ref{app:other-publications} lists an additional seven
co-authored publications. Among others, \citet{Misof2014} (page
\pageref{app:Misof2014}) consummates the inference of a robust
phylogenetic backbone tree of 145 insect species from all major insect
orders. I was a major contributor to the study. The analysis includes
node dating with a comprehensive calibration dataset of 37 fossils. The
study presents a semi-automated and documented reproducible workflow
that facilitates phylogeny inference from transcriptomic data of many
species. It also provides the backbone phylogeny and node dates for the
study in chapter \ref{cha:dynamics} and the breeding ground for the
development of Orthograph (chapter \ref{cha:orthograph}).
